util/pom.xml

```diff
@@ -88,6 +87,7 @@
                     </dependency>
                 </dependencies>
             </plugin>
+-->
```

Comment:
It is commented out now. We need to create a story to upgrade checkstyle that understands java 8 syntax